mentoring/frontend/templates/frontend/root.html

```diff
@@ -4,25 +4,14 @@
   <meta charset="utf-8">
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
   <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width" />
+  <meta property="csp-nonce" content="{{ request.csp_nonce }}">
 </head>
 <body>
 <div id="app">
     <!-- React will load here -->
 </div>
 </body>
-{% csrf_token %}
```

Comment:
This might be required to generate a CSRF token on the initial page request. You might try adding it again if forms fail. But, I did see a ``csrftoken`` cookie in the ``PUT`` request, so maybe it is only needed to make ``document.querySelector('[name=csrfmiddlewaretoken]').value`` work.

Reply:
With a proper token in ``settings.js``, this is not needed.